wagtail/admin/tests/tests.py

```diff
@@ -31,6 +31,15 @@ def test_simple(self):
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "Test Site")
 
+    def test_accessible_skip_link_exists(self):
+        response = self.client.get(reverse("wagtailadmin_home"))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(
+            response,
+            '<a class="skiplink button" href="#main" data-controller="w-focus" data-action="w-focus#focus:prevent">Skip to main content</a>',
+        )
+        self.assertContains(response, ' id="main">')
```

Comment:
Would it be better to use BeautifulSoup here so we don't have to assert `class="skiplink button"` and can actually select an element with `id="main"`?

Reply:
Good idea, done.